lib/SQL/Translator/Parser/MySQL.pm

```diff
@@ -326,7 +326,7 @@ create : CREATE PROCEDURE NAME not_delimiter "$delimiter"
 PROCEDURE : /procedure/i
     | /function/i
 
-create : CREATE or_replace(?) create_view_option(s?) /view/i NAME /as/i view_select_statement "$delimiter"
+create : CREATE or_replace(?) create_view_option(s?) /view/i NAME parens_field_list(?) /as/i view_select_statement "$delimiter"
```

Comment:
Sorry, I misread your statement, ignore the previous comment.  It'd be easier to just store the fields in the `%views` hash and use that if present instead of the alias list in the `->add_view` call in `sub parse { … }`.